app/enssubdomain/models.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can you add the copyright boilerplate to this file?

Reply:
done